modules/MIPS/clinical_measures.php

```diff
@@ -530,9 +424,6 @@ function Form_Validate() {
               <td style="text-align:center"><?php echo $row['excluded']; ?></td>
 <?php
             }
-
-
- 
```

Comment:
Surely the editor stuff here

Reply:
@muarachmann Do i need to make any changes ?